modules/ting/src/Search/BooleanStatementInterface.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
```

Comment:
Namespaced classes, interfaces and traits should not begin with a file doc comment

Reply:
This is true for all the classes, will not write it again :-)